src/System.IO.Pipelines/System.IO.Pipelines.sln

```diff
@@ -30,8 +30,8 @@ Global
 		{9E984EB2-827E-4029-9647-FB5F8B67C553}.Debug|Any CPU.Build.0 = netcoreapp-Debug|Any CPU
 		{9E984EB2-827E-4029-9647-FB5F8B67C553}.Release|Any CPU.ActiveCfg = netcoreapp-Release|Any CPU
 		{9E984EB2-827E-4029-9647-FB5F8B67C553}.Release|Any CPU.Build.0 = netcoreapp-Release|Any CPU
-		{1032D5F6-5AE7-4002-A0E4-FEBEADFEA977}.Debug|Any CPU.ActiveCfg = netcoreapp-Debug|Any CPU
-		{1032D5F6-5AE7-4002-A0E4-FEBEADFEA977}.Debug|Any CPU.Build.0 = netcoreapp-Debug|Any CPU
+		{1032D5F6-5AE7-4002-A0E4-FEBEADFEA977}.Debug|Any CPU.ActiveCfg = netstandard-Debug|Any CPU
```

Comment:
What does this change do exactly? Do we need it?

Reply:
Not sure its needed.